packages/p2p-v2/src/pages/orders/components/ChatFooterIcon/ChatFooterIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Button } from '@deriv-com/ui';
+import AttachmentIcon from '../../../../public/ic-attachment.svg';
+import SendMessageIcon from '../../../../public/ic-send-message.svg';
+
+type TChatFooterIconProps = {
+    length: number;
+    onClick: () => void;
```

Comment:
Does this work @nada-deriv? 🤔  
```suggestion
    onClick: MouseEventHandler<HTMLButtonElement>;
```